pgjdbc/src/test/java/org/postgresql/test/jdbc2/UpdateableResultTest.java

```diff
@@ -40,6 +40,10 @@ public void setUp() throws Exception {
     TestUtil.createTable(con, "updateable",
         "id int primary key, name text, notselected text, ts timestamp with time zone, intarr int[]");
     TestUtil.createTable(con, "second", "id1 int primary key, name1 text");
+    TestUtil.createTable(con, "serialtable", "gen_id serial primary key, name text");
+    TestUtil.createTable(con, "compositepktable", "gen_id serial, dec_id serial, name text");
```

Comment:
Can you put the name column between the two pk's just to make it so they aren't together?